src/wallet.ts

```diff
@@ -0,0 +1,218 @@
+import ethTx = require('ethereumjs-tx');
+import ethWallet = require('ethereumjs-wallet');
+
+declare const Buffer;
+declare const setTimeout;
+
+export default class Wallet {
+  length: number;
+  nonce: number;
+  web3: any;
+
+  constructor(web3: any) {
+    this.length = 0;
+    this.nonce = 0;
+    this.web3 = web3;
+  }
+
+  _findSafeIndex(pointer = 0) {
+    pointer = pointer;
```

Comment:
Is there a reason for this assignment? Can't we just use the `pointer` variable as-is?

Reply:
I think the idea is that we will never overwrite a wallet, in case of some weird code getting through and trying to do that.

Reply:
Yes I think we can just use the pointer variable. No reason for this assignment as far as I can tell.